crates/cli/src/subcommands/sql.rs

```diff
@@ -160,27 +160,12 @@ pub(crate) async fn run_sql(builder: RequestBuilder, sql: &str, with_stats: bool
 fn stmt_result_to_table(stmt_result: &StmtResultJson) -> anyhow::Result<(StmtStats, tabled::Table)> {
     let stats = StmtStats::from(stmt_result);
     let StmtResultJson { schema, rows, .. } = stmt_result;
-
-    let mut builder = tabled::builder::Builder::default();
-    builder.set_header(
-        schema
-            .elements
-            .iter()
-            .enumerate()
-            .map(|(i, e)| e.name.clone().unwrap_or_else(|| format!("column {i}").into())),
-    );
-
     let ty = Typespace::EMPTY.with_type(schema);
-    for row in rows {
-        let row = from_json_seed(row.get(), SeedWrapper(ty))?;
-        builder.push_record(
-            ty.with_values(&row)
-                .map(|value| satn::PsqlWrapper { ty: ty.ty(), value }.to_string()),
-        );
-    }
 
-    let mut table = builder.build();
-    table.with(Style::psql());
+    let table = build_table(
+        schema,
+        rows.iter().map(|row| from_json_seed(row.get(), SeedWrapper(ty))),
+    )?;
```

Comment:
why was this stuff changed? it's not obvious to me from the PR description.

Reply:
...Or maybe not. Sigh.

Reply:
Ok, I had to rebase and force-push, but this should be fixed.